### PR TITLE
Release 0.4.0 (master)

### PR DESCRIPTION
**Pull request by TheAdamBorek**
It's a final PR for the release 0.4.0.

The checklist

- [x] find a fun name for the release -> Reiwa Bento
- [ ] merge the PR to master
- [ ] create a tag on the last commit
- [ ] release Bento 0.4.0 to CocoaPods
- [ ] create a new release in the release tab
- [ ] merge the PR to develop (PR #161) 

Let me know if I've forgotten about something

### CODE REVIEW

**Comment by TheAdamBorek** [comment]
How do we call next release? :D  https://github.com/Babylonpartners/Bento/releases
I'm bad at fish names 🙃 
Blowfish Bento? 😅 ☠️

**Comment by AliSoftware** [comment]
> How do we call next release? :D https://github.com/Babylonpartners/Bento/releases
> I'm bad at fish names 🙃


It's not only fish names, for example "Marukonouchi " is not a fish but the typo of Bento (typically served during kabuki intermissions) 😉 

We could call this one "Koraku Bento" for example: [inspiration names here](https://gurunavi.com/en/japanfoodie/2017/10/japanese-bento-boxes-types-ingredients.html)

**Comment by TheAdamBorek** [comment]
👍 for Koraku Bento ;)

**Comment by TheAdamBorek** [comment]
but... I think @inamiy should be responsible for names here 😂

**Comment by inamiy** [comment]
I'm +1 for " ㊗️  Reiwa Bento " 🎉 
http://www.tobu-dept.jp/funabashi/event/detail/2117

File: Bento.podspec [hunk 1]
```diff
@@ -1,21 +1,23 @@
 Pod::Spec.new do |s|
 
-  s.name          = "Bento"
-  s.version       = "0.3"
-  s.summary       = "Component based abstraction on top of UITableView and UICollectionView"
+  s.name             = "Bento"
+  s.version          = "0.4"
```
Comment (TheAdamBorek) [inline Bento.podspec]: This is my change

File: BentoKit.podspec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name          = "BentoKit"
-  s.version       = "0.3"
+  s.version       = "0.4"
```
Comment (TheAdamBorek) [inline BentoKit.podspec]: This is my change

**Comment by viorel15** [comment]
Any new additions on this PR or everything has been already reviewed in separate PR's?

**Comment by TheAdamBorek** [comment]
The PR has many other changes because master wasn't updated for some time.

I've just updated version in `podspec` (mentioned in comments) https://github.com/Babylonpartners/Bento/pull/160/commits/37568894c8b39aee36e750f20cea5030db1ae139

**Comment by mluisbrown** [comment]
LGTM 🎉  Pity we didn't remove TitledDescription from BentoKit though 😞

**Review by mluisbrown — APPROVED** [review]

**Comment by TheAdamBorek** [comment]
@mluisbrown I just want to release this 0.4.0 ASAP. 
We can remove TitledDescription and release 0.4.1. quickly :)

**Review by viorel15 — APPROVED** [review]

**Pull request merged**